CMakeModules/CPackConfig.cmake

```diff
@@ -101,5 +101,9 @@ SET(CPACK_SOURCE_IGNORE_FILES
 # Ignore build directories that may be in the source tree
 FILE(GLOB_RECURSE CACHES "${CMAKE_SOURCE_DIR}/CMakeCache.txt")
 
+SET(CPACK_WIX_LICENSE_RTF "${PROJECT_SOURCE_DIR}/LICENSE.rtf")
+SET(CPACK_WIX_PATCH_FILE "${PROJECT_SOURCE_DIR}/wix/WIXPatch.wxs")
+SET(CPACK_WIX_UPGRADE_GUID "FF9E2D77-CDC7-4D24-8B7B-99D66EDEE862")
```

Comment:
It is indeed randomly generated, and if you wanted to have multiple ArrayFire installations side by side, then this number could change.  Otherwise, leaving this number intact allows for future versions of ArrayFire to replace existing versions on the system.